crates/containerd-shim-wasm/src/sandbox/containerd/client.rs

```diff
@@ -501,20 +484,22 @@ impl Client {
                 log::debug!(
                     "updating image content with precompile digest to avoid garbage collection"
                 );
-                let mut image_content = self.get_info(&image_digest)?;
+                let mut image_content = self.get_info(&image_digest).await?;
                 image_content.labels.insert(
                     format!("containerd.io/gc.ref.content.precompile.{}", i),
-                    precompiled_content.digest,
+                    precompiled_content.digest.clone(),
```

Comment:
why is there a extra clone() here?

Reply:
It was there so that I could do `precompiled_content.release().await` later.
without the clone `precompiled_content` would be partially destructured, and I wouldn't be able to call methods on it.

But instead of doing `precompiled_content.release()` I can do `precompiled_content.lease.release()` and save the clone, indeed :-)
Thanks!